extensions/deployers/licenses_deploy.py

```diff
@@ -0,0 +1,11 @@
+from conan.tools.files import copy
+import os
+
+
+def deploy(graph, output_folder):
```

Comment:
Good as a start. In the future, I'd make it more complete, being able to report the licenses from teh recipe attributes, and report over skipped binaries (not necessary part of the current product)

Reply:
I left the deployers for another PR to focus on the command